Services/DataBaseService/Repositories/UserRepository.cs

```diff
@@ -101,22 +119,45 @@ public void EditUser(User user, PasswordHashChangeRequest password)
                     {
                         if (dbUserCredential.PasswordHash != password.OldPasswordHash)
                         {
-                            throw new ForbiddenException("Can't change password, old password is wrong");
+                            var exception = new ForbiddenException("Can't change password, old password is wrong");

```

Comment:
May be we can create it in throwing, just logging before the throw?

Reply:
What is the difference?

Reply:
> What is the difference?

What do you need variable `exception` for?